docker/gcp_pudl_etl.sh

```diff
@@ -210,7 +211,7 @@ ETL_SUCCESS=0
 SAVE_OUTPUTS_SUCCESS=0
 UPDATE_NIGHTLY_SUCCESS=0
 UPDATE_STABLE_SUCCESS=0
-DATASETTE_SUCCESS=0
+# DATASETTE_SUCCESS=0
```

Comment:
totally non-blocking: I sort of think we should just delete the datasette code instead of commenting it out. If we want to turn it back on we can revert this commit. Would be kind of cleaner to do that if this PR was *only* disabling datasette instead of disabling datasette *and* bash style changes but :shrug:

Reply:
Oh. I didn't even realize my bash linter had done all that. Let me revert.